tests/try.spec.ts

```diff
@@ -353,4 +353,12 @@ describe("all", () => {
 
         expect(result.get).toThrow("expected failure 1");
     });
-});
\ No newline at end of file
+});
+
+describe("void", () => {
+    it("returns a success of undefined with void type", () => {
+        const result = Try.void();
+        expect(result.isSuccess()).toEqual(true);
+        expect(result.map((r) => r)).toBeUndefined();
```

Comment:
its still a `Try`?

```suggestion
        expect(result.get()).toBeUndefined();
```

Reply:
heh yeah... the problem was the linter was complaining about `.get()` on a void...

Reply:
oh whoops, forgot the get 😓

Reply:
meh seems to be fine, updated